proto/rill/runtime/v1/api.proto

```diff
@@ -309,6 +309,8 @@ message Instance {
   // If true, the runtime will store the instance's catalog in its OLAP store instead
   // of in the runtime's metadata store. Currently only supported for the duckdb driver.
   bool embed_catalog = 6;
+  // instance specific enviornment variables
+  map<string, string> env = 7;
 }
```

Comment:
Also add `project_env` here? (Allows us to show env defaults and env overrides in a listing in UI/CLI)